ci/test_examples.sh

```diff
@@ -1,3 +1,3 @@
-for example in $(find ./examples/ -iname *.py); do
+for example in $(find ./examples/ -iname *.py) $(find ../docs/examples/ -iname *.py); do
```

Comment:
```suggestion
for example in $(find ./examples/ -iname *.py) $(find ./docs/examples/ -iname *.py); do
```

Since this is run from the root directory of the repository via `source ci/test_examples.sh`, it makes sense to keep the path relative to that.

Reply:
Ops. My bad..